manifests/init.pp

```diff
@@ -55,6 +56,12 @@
     $dhcp_interfaces = $interfaces
   }
 
+  if $dnsupdatekey {
+    $dnsupdatekey_split    = split($dnsupdatekey, '[/]')
```

Comment:
best practices dictates that we should try and use $_ prefix for private variables that are not parameters, it makes it easier to maintain, also this will fail with strict variables